index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Apple</title>
+    <link href="https://g1.nyt.com/fonts/css/web-fonts.5810def60210a2fa7d0848f37e3fa048bb6147b1.css" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" />
+    <link rel="stylesheet" href="assets/styles/reset.css">
+    <link rel="stylesheet" href="assets/styles/main.css">
+    </head>
+    <body>
+        <main >
+                <div class="page-top flex-v">
+                        <nav>
+                            <ul class="header-links flex-h">
+                            <li class="flex-1fr text-align-center"><a href="#" ><i class="fab fa-apple fa-2x light-gray"></i></a></li>
+                            <li class="flex-1fr text-align-center"><a href="#">Store</a></li>
+                            <li class="flex-1fr text-align-center"><a href="#" >Mac</a></li>
+                            <li class="flex-1fr text-align-center"><a href="#" >iPod</a></li>
+                            <li class="flex-1fr text-align-center"><a href="#">iPhone</a></li>
+                            <li class="flex-1fr text-align-center"><a href="#" >iPad</a></li>
+                            <li class="flex-1fr text-align-center"><a href="#" >iTunes</a></li>
+                            <li class="flex-1fr text-align-center"><a href="#" >Support</a></li>
+                            <li class="flex-1fr text-align-center">
+                                <input type="search" name="search" id="search"><i class="fas fa-search icon-inside-input"></i></li>
+                            </ul>
+                        </nav>
+                        <div class="titles-under-nav">
+                            <h1>
+                              <img src="./assets/images/your_verse_title_small.png" alt="What will you verse be">
+                            </h1>
+                            <h2>
+                              <img src="./assets/images/your_verse_subhead_small.png" alt="everyone ...">
```

Comment:
- [ ] You've done well requesting this change, however, you should make sure to include the reason as per why the change is required, for example: `To improve the accessibility of your wepga,e it would be better to add meaningful alt tags to your images`